MixedRemoteViewCompositor/Samples/LowLatencyMRC/UWP/Viewer/Capture.xaml.cs

```diff
@@ -48,7 +48,7 @@ public Capture()
 
         private async void StartCapture()
         {
-            captureEngine = await CaptureEngine.CreateAsync(false);
+            captureEngine = await CaptureEngine.CreateAsync(true);

```

Comment:
change function signature to pass in the bool value for audio

Reply:
Sorry for the late response. Just got a chance to revisit this project.  Added a checkbox in the UI and changed the signature of StartCapture. Please check the latest commit.